test/IDE/signature_help_closure_param.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-ide-test -signature-help -code-completion-token=CLOSURE_PARAM -source-filename=%s | %FileCheck %s --check-prefix=CLOSURE_PARAM
+
+func apply<Value, Result>(value: Value, body: (Value) -> Result) -> Result {
+  return body(#^CLOSURE_PARAM^#)
+}
+
+// CLOSURE_PARAM:      Begin signatures, 1 items
+// CLOSURE_PARAM-NEXT: Signature[Active]: body(<param active>Value</param>) -> Result
+// CLOSURE_PARAM-NEXT: End signatures
```

Comment:
Switched back to `FileCheck` from `diff` for `swift-ide-test` tests since the output is much nicer now besides that `swift-ide-test` prints `"found code completion token <TOKEN> at offset OFFSET>"` which we don't want to match with diff.